readme.md

```diff
@@ -2,11 +2,16 @@
 
 [![Build Status](https://travis-ci.com/brumik/react-wooden-tree.svg?branch=master)](https://travis-ci.com/brumik/react-wooden-tree)
 
-A simple solution to display hierarchical structures (Tree View) while using React.
+The tree is bit more complex in the new version. You can use it with
+redux and of course without it. 
+
+For detailed information visit the docs page: (and turn off inherited methods)
+[https://brumik.github.io/react-wooden-tree/](https://brumik.github.io/react-wooden-tree/).
 
 ## Dependencies
-To use the Wooden Tree two requirements must met.
+To use the Wooden Tree you need two dependencies:
```

Comment:
This is not needed at all, the list is sufficient and the number is invalid anyway.